README.md

```diff
@@ -796,14 +796,7 @@ The `size_t malloc_object_size(void *ptr)` function returns an *upper bound* on
 the accessible size of the relevant object (if any) by querying the malloc
 implementation. It's similar to the `__builtin_object_size` intrinsic used by
 `_FORTIFY_SOURCE` but via dynamically querying the malloc implementation rather
-than determining constant sizes at compile-time. The current implementation is
```

Comment:
It's still not fully accurate for large allocations. I'd just leave this documentation alone and handle this in a separate PR.